### ISSUE TITLE
Tailor shop

### ISSUE THREAD

**Issue by ravenbuilder934** [ldtteam/minecolonies-features#167]
### Prerequisites 

- [x] I am running the latest **alpha** version of Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my feature does not already exist.
- [x] I made sure that **this feature request is not a duplicate of any existing feature request.**

### Context 

- Minecraft Version: 1.15.2
- MineColonies Version: MineColonies Official pack

## Is your feature request related to a problem?
No

## Feature description
In my opinion it's kinda weird that colonists' clothes/skins change without them actually getting new clothes. It would be great if there was a mid to late game building, called the tailor shop, where colonists would go to get new clothes. The tailor would 'create' the clothes with leather, string, wool, dye, shears (scissors), and maybe a couple iron nuggets for needles. Colonists would get a happiness boost when they got new clothes.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the feature request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by therealpro101** [comment]
I agree there should be a tailored system but I would further expand on it by having certain debuffs related to the weather/biome and buffs related to clothes they wear in certain biomes and make them where you start with clothes at the beginning then it slowly evolves to you should build a tailor it would bring a certain difficulty to the mayor. If applicable since there is a nether colony it could further expand to the nether and make the nether hot. All these options are configurable and they would bring a sense of vibrance to the game. Not only would it increase the features, but it would envision ray's image of the age of empires kinda like an old game called banished.

**Comment by Bossos11** [comment]
maybe make it a prerequisite for a worker of a higher level to have specific clothing(level)
